tests/tests.rs

```diff
@@ -1354,3 +1354,186 @@ fn cmd_lens_run() {
 
     rls.shutdown(rls_timeout());
 }
+
+#[test]
+fn test_find_definitions() {
+    const SRC: &str = r#"
+        struct Foo {
+        }
+
+        impl Foo {
+            fn new() {
+            }
+        }
+
+        fn main() {
+            Foo::new();
+        }
+    "#;
+
+    let p = project("simple_workspace")
+        .file("Cargo.toml", &basic_bin_manifest("bar"))
+        .file("src/main.rs", SRC)
+        .build();
+
+    let root_path = p.root();
+    let mut rls = p.spawn_rls();
+
+    rls.request(
+        0,
+        "initialize",
+        Some(json!({
+            "rootPath": root_path,
+            "capabilities": {},
+            "initializationOptions": {
+                "settings": {
+                    "rust": {
+                        "racer_completion": false
+                    }
+                }
+            }
+        })),
+    )
+    .unwrap();
+
+    rls.wait_until_done_indexing(rls_timeout());
+
+    let uri = format!("file://{}/src/main.rs", root_path.display());
```

Comment:
Windows tests fail:
Got
```
"file:///C:/Users/travis/build/rust-lang/rls/target/rlsit/t13/simple_workspace/src/main.rs"
```

Expected
```
file://C:\\Users\\travis\\build\\rust-lang\\rls\\target\\rlsit\\t13\\simple_workspace/src/main.rs
```

I think something like `let uri = Url::from_file_path(p.root().join("src/main.rs")).unwrap();` should make it work cross-platform